Header header
float64 x
float64 theta
float64 vel
bool rightLim
bool leftLim
